src/app/middleware/localStorageMiddleware.ts

```diff
@@ -1,3 +1,4 @@
+import { Middleware } from 'redux';
```

Comment:
Question, since in the description you say we're removing this package too, what happens to it's usage in such instances? We create our own types or does RTK offer these too?

Reply:
RTK packs redux inside it. So we import from the toolkit instead of core  redux

Reply:
Oh, noice!